docs/src/main/asciidoc/transaction-guide.adoc

```diff
@@ -166,6 +166,21 @@ public class SantaClausService {
 You cannot use `UserTransaction` in a method having a transaction started by a `@Transactional` call.
 ====
 
+== Configuring the transaction timeout
+You can configure the default transaction timeout, the timeout that will applies to all transactions managed by the transaction manager, via the property `quarkus.transaction-manager.default-transaction-timeout`, specified as a duration.
+
+[NOTE]
+====
+The format for durations uses the standard `java.time.Duration` format.
+You can learn more about it in the link:https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#parse-java.lang.CharSequence-[Duration#parse() javadoc].
+
+You can also provide duration values starting with a number.
+In this case, if the value consists only of a number, the converter treats the value as seconds.
+Otherwise, `PT` is implicitly appended to the value to obtain a standard `java.time.Duration` format.
+====
```

Comment:
maybe we should have an include for that one? I think we already have it in several places.

`duration-format-note.adoc`

Reply:
Note: this part can be done in a follow-up commit, let's focus on what is important here.

Reply:
There is no duration-format-note.adoc. 
I copy/paste the not from the Datasource documentation.

Reply:
Yes, sorry, I wasn't clear: it was about creating it to avoid duplication as I suspect we will need to have this note in several files (it's probably already the case).

But let's do that in another PR, once this one is in.

Reply:
I'll make a PR about the `duration-format-note.adoc` :-)